src/components/Contributors/styles.module.css

```
@@ -0,0 +1,30 @@
+.avatarWrapper {
+  display: flex;
+}
+@media (min-width:1025px) {
+  .avatar {
+    min-width: 80px;
+    border-radius: 40px;
+    max-height: 80px;
+    margin-left: 10px;
+  }
+
+}
+
+@media (min-width:320px) {
+  .avatar {
+    width: 80px;
+    border-radius: 40px;
+    margin-left: 10px;
+    margin-top: 10px;
+  }
+}
+
+@media (max-width:320px) {
+  .avatar {
+    width: 50px;
+    border-radius: 40px;
+    margin-left: 10px;
+    margin-top: 10px;
+  }
+}
```

Propriedades que não variam nos media-queries poderiam ficar no início, fora delas.
```suggestion
.avatarWrapper {
  display: flex;
  border-radius: 40px;
  margin-left: 10px;
  margin-top: 10px;
}
@media (min-width:1025px) {
  .avatar {
    min-width: 80px;
    max-height: 80px;
  }
}

@media (min-width:320px) {
  .avatar {
    width: 80px;
  }
}

@media (max-width:320px) {
  .avatar {
    width: 50px;
  }
}
```